src/components/Modal/FormModal.tsx

```diff
@@ -0,0 +1,39 @@
+import { Modal } from '@mui/material'
+import { Box } from '@mui/system'
+import Button from '@mui/material/Button'
+
+export const FormModal: React.FC<{
+  openAddModal: boolean,
+  handleCloseAddModal: () => void
+  header: string
+}> = ({ openAddModal, handleCloseAddModal, header }) => {
+
+  return (
+    <Modal data-testid="add-modal-id" open={openAddModal} onClose={handleCloseAddModal}>
+      <Box className="modal-box">
+        <h2>{header}</h2>
+        <div className='form'>
+          <form data-testid="form-id">
+            <div className='form-row'>
+              <label>Name:</label>
+              <input type='text' placeholder='Enter your name here'/>
```

Comment:
doesn't have to be in this PR, but I believe adding `htmlFor` (aka `for`) on the label can help with accessibility.

HTML `for` for `<label>`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes

Accessibility:
https://www.w3.org/WAI/tutorials/forms/labels/